src/postgres/event-handlers/CommentCreated.ts

```diff
@@ -1,35 +1,39 @@
-import { CommentId } from '@subsocial/types/substrate/interfaces/subsocial';
+import { Post, PostId } from '@subsocial/types/substrate/interfaces/subsocial';
 import { substrate } from '../../substrate/subscribe';
 import { insertCommentFollower } from '../insert-follower';
 import { insertActivityComments, insertActivityForComment } from '../insert-activity';
 import { fillNotificationsWithAccountFollowers, fillNotificationsWithPostFollowers } from '../fill-activity';
 import { substrateLog as log } from '../../connections/loggers';
-import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
+import { SubstrateEvent } from '../../substrate/types';
+import { VirtualEvents } from '../../substrate/utils';
 
-export const onCommentCreated: EventHandlerFn = async (eventAction: SubstrateEvent) => {
+export const onCommentCreated = async (eventAction: SubstrateEvent, post: Post) => {
   const { data } = eventAction;
   await insertCommentFollower(data);
-  const commentId = data[1] as CommentId;
+  const commentId = data[1] as PostId;
 
-  const comment = await substrate.findComment(commentId);
-  if (!comment) return;
+  const {
+    extension: { asComment: { root_post_id, parent_id } },
+    created: { account }
```

Comment:
You don't need to get `account` from comment, because we already have a comment author in blockchain event.